configs/reconciler.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 db:
-  driver: postgres
+  driver: sqlite
```

Comment:
In default case will we use `sqllite`?

Reply:
at least for local development it's simpler to use. For CI pipelines we should think whether we stay with SQLite or run them with Postgres (or maybe both.. but depends on the execution time / resource consumption).